lib/index.ts

```diff
@@ -32,7 +32,33 @@ function createListrOptions (options) {
   }
 }
 
-async function runContentfulImport (params) {
+// These type definitions follow what is specified in the Readme
+type RunContentfulImportParams = {
+  spaceId: string,
+  environmentId?: string,
+  managementToken: string,
+  contentFile?: string,
+  content?: object,
+  contentModelOnly?: boolean,
+  skipContentModel?: boolean,
+  skipLocales?: boolean,
+  skipContentPublishing?: boolean,
+  uploadAssets?: boolean,
+  assetsDirectory?: string,
+  host?: string,
+  proxy?: string,
+  rawProxy?: string,
+  rateLimit?: number,
+  headers?: object,
```

Comment:
Maybe we can be more specific here
```suggestion
  headers?: [header: string]: string | string[] | undefined,
```

Reply:
that's a good idea! the shape is a bit different tho, I will update it
https://github.com/contentful/contentful-import/blob/feat/adjust-types-for-public-interface/lib/utils/headers.ts#L13